aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/tests/DataSetupExtension.java

```diff
@@ -55,6 +56,22 @@ private void importDefinitions(AATHelper aat, String s2sToken) {
             .post("/import");
     }
 
+    private void importNewDefinitionsForAMIntegration(AATHelper aat, String s2sToken) {
+        AuthenticatedUser importer = aat.getIdamHelper()
+            .authenticate(aat.getImporterAutoTestEmail(),
+                aat.getImporterAutoTestPassword());
+
+        RestAssured.given()
```

Comment:
just add this new import call inside the importDefinitions method as a second upload call. It introduces vagueness to the code to have 2 method names that are similar but not clear how different. besides, the new import is not for am integration only, but it sounds like that. so, please merge these methods under the original importDefinitions method. You can add a line of comment for clarity if needed inside that method.